<?xml version="1.0"?>

<launch>
    <node name="lift_driver" pkg="lift_driver" type="lift_driver" respawn="false" output="screen" >
        <param name="serial_name" type="string" value="/dev/ttyUSB0" />
    </node>
</launch>
